components/more-less/more-less.js

```diff
@@ -234,13 +237,27 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 		this.expanded = true;
 	}
 
-	__focusIn() {
+	__focusIn(e) {
 		if (this.inactive || this.expanded) {
 			return;
 		}
 
-		this.__expand();
-		this.__autoExpanded = true;
+		const target = e.composedPath()[0];
+
+		if (isSafari) {
+			target.scrollIntoViewIfNeeded();
+		}
```

Comment:
Safari doesn't immediately scroll the focused element into view within the component content div, so we force it to. This still seems better than doing a bunch of custom measurement to determine if the target is on screen etc.